code/src/Components/MovieList.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import './movie-list.css'
+
+export const MovieList = () => {
+  const [movies, setMovies] = useState([])
+
+  useEffect(() => {
+    fetch('https://api.themoviedb.org/3/movie/popular?api_key=078bf3a954eadf3e9528b49c2a6a9682&language=en-US&page=1')
+      .then((res) => res.json())
+      .then((json) => {
+        setMovies(json.results)
+      })
+  }, [])
+
+  return (
+    <div className="movie-list">
+      {movies.map((movie) => (
+        <div key={movie.id} className="movie-poster">
+          <Link className="movie-link" to={`/movie/${movie.id}`}>
+            <img src={`https://image.tmdb.org/t/p/w500${movie.poster_path}`} alt={movie.title} />
+            <div className="poster-details">
```

Comment:
You have chosen great classNames! Makes it east to follow and to get a vision of what the site contains 👍🏼